index.html

```diff
@@ -3401,8 +3420,78 @@ <h2>
               The user chooses a new shipping option.
             </td>
           </tr>
+          <tr>
+            <td>
+              <code><dfn>paymentmethodchange</dfn></code>
+            </td>
+            <td>
+              <a>PaymentMethodChangeEvent</a>
+            </td>
+            <td>
+              The user chooses a different <a>payment method</a> within a
+              <a>payment handler</a>.
+            </td>
+          </tr>
         </table>
       </section>
+      <section data-dfn-for="PaymentMethodChangeEvent" data-link-for=
+      "PaymentMethodChangeEvent">
+        <h2>
+          <dfn>PaymentMethodChangeEvent</dfn> interface
+        </h2>
+        <pre class="idl">
+          [Constructor(DOMString type, optional PaymentMethodChangeEventInit eventInitDict), SecureContext, Exposed=Window]
+          interface PaymentMethodChangeEvent : PaymentRequestUpdateEvent {
+            readonly attribute DOMString methodName;
+            readonly attribute object? methodDetails;
+          };
+        </pre>
+        <section>
+          <h2>
+            <dfn>methodDetails</dfn> attribute
+          </h2>
+          <p data-link-for="PaymentMethodChangeEventInit">
+            When getting, returns the value it was initialized with. See
+            <a>methodDetails</a> member for more information.
```

Comment:
Here and below, "member of PaymentMethodChangeEventInit" perhaps? Otherwise before you click on it it looks like it'll just link to itself.